### ISSUE TITLE
[Request] remote dev support

### ISSUE THREAD

**Issue by jacobcoughenour** [iCrawl/discord-vscode#257]
Problem:
When working in a remote vscode environment (insider build feature), the extension loads on the remote machine instead of the local machine.  This causes the discord rpc connection to fail since it is on a different machine.

Workaround:
Add this to user settings.json
```
"remote.extensionKind": {
	"icrawl.discord-vscode": "ui"
},
```
then reload and the extension works again.

Potential Fix:
Add `"extensionKind": "ui"` to package.json to load the extension locally by default.

https://code.visualstudio.com/api/advanced-topics/remote-extensions#incorrect-execution-location

**Comment by iCrawl** [comment]
I'll get into fixing that ASAP.

**Comment by iCrawl** [comment]
Released in version 3.6.0
